webpack/__mocks__/foremanReact/redux/actions/toasts.js

```diff
@@ -1,5 +1,5 @@
 export const addToast = toast => ({
```

Comment:
This file should be renamed to `webpack/__mocks__/foremanReact/components/ToastsList/index`

Reply:
:facepalm: sure, done.